scripts/foundry/contract-upgrades/2.1.0/ExecuteNitroContracts2Point1Point0Upgrade.s.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.16;
+
+import "forge-std/Script.sol";
+import {
+    NitroContracts2Point1Point0UpgradeAction,
+    ProxyAdmin
+} from "../../../../contracts/parent-chain/contract-upgrades/NitroContracts2Point1Point0UpgradeAction.sol";
+import {IBridge} from "@arbitrum/nitro-contracts-2.1.0/src/bridge/IBridge.sol";
+import {IRollupCore} from "@arbitrum/nitro-contracts-2.1.0/src/rollup/IRollupCore.sol";
+import {IUpgradeExecutor} from "@offchainlabs/upgrade-executor/src/IUpgradeExecutor.sol";
+
+/**
+ * @title ExecuteNitroContracts1Point2Point1UpgradeScript
+ * @notice This script executes nitro contracts 2.1.0 upgrade through UpgradeExecutor
+ */
+contract ExecuteNitroContracts2Point1Point0UpgradeScript is Script {
+    function run() public {
```

Comment:
before execute, check the current wasm root matches `DeployNitroContracts2Point1Point0UpgradeActionScript.COND_WASM_MODULE_ROOT` (v20)

Reply:
https://github.com/OffchainLabs/orbit-actions/pull/16/commits/9511bc7a3bfe21b179549adcb737476a10bc4ee9